src/ostree/ot-admin-builtin-status.c

```diff
@@ -31,16 +31,25 @@
 static gboolean opt_verify;
 static gboolean opt_skip_signatures;
 static gboolean opt_is_default;
+static gboolean opt_booted_index;
+static gboolean opt_pending_index;
+static gboolean opt_rollback_index;
 
-static GOptionEntry options[]
-    = { { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
-          NULL },
-        { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures,
-          "Skip signatures in output", NULL },
-        { "is-default", 'D', 0, G_OPTION_ARG_NONE, &opt_is_default,
-          "Output \"default\" if booted into the default deployment, otherwise \"not-default\"",
-          NULL },
-        { NULL } };
+static GOptionEntry options[] = {
+  { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status",
+    NULL },
+  { "skip-signatures", 'S', 0, G_OPTION_ARG_NONE, &opt_skip_signatures, "Skip signatures in output",
+    NULL },
+  { "is-default", 'D', 0, G_OPTION_ARG_NONE, &opt_is_default,
+    "Output \"default\" if booted into the default deployment, otherwise \"not-default\"", NULL },
+  { "booted-index", 'b', 0, G_OPTION_ARG_NONE, &opt_booted_index,
+    "Output \"default\" if booted into the default deployment, otherwise \"not-default\"", NULL },
+  { "pending-index", 'p', 0, G_OPTION_ARG_NONE, &opt_pending_index,
+    "Output \"default\" if booted into the default deployment, otherwise \"not-default\"", NULL },
```

Comment:
So the:

"Output \"default\" if booted into the default deployment, otherwise \"not-default\""

shoudl have something that actually makes sense for these options

Reply:
Apologies - should have picked up on this :facepalm: 

Have added some descriptors that make sense now, but let me know if you want these changing.

Have squashed my commits into one as most projects ask for this anyway, therefore cannot point you to the commit with this fix.